src/vm/jitinterface.cpp

```diff
@@ -12633,6 +12633,13 @@ CORJIT_FLAGS GetCompileFlags(MethodDesc * ftn, CORJIT_FLAGS flags, CORINFO_METHO
         flags.Clear(CORJIT_FLAGS::CORJIT_FLAG_DEBUG_INFO);
     }
 
+#ifndef CROSSGEN_COMPILE
+    if (g_pConfig->ForceQuickJit())
+    {
+        flags.Add(CORJIT_FLAGS::CORJIT_FLAG_TIER0);
+    }
+#endif
+
     return flags;
```

Comment:
Could we end up with both `Tier0` and `Tier1` set for some methods?

Reply:
If `ForceQuickJit` is set it also disables tiering, so that shouldn't be possible